### DISCUSSION TITLE
modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/JSDoc

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#67] [Announcements]
# modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/JSDoc

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/JSDoc

**Comment by M1hono** [comment]
JSDoc在函数/轮子的应用上也很广泛，我会在之后将这一部分添入本章节或归入进阶内容。

基本上，该章节是你从入门步向进阶的**必要知识**
